client.go

```diff
@@ -190,6 +190,26 @@ func (c *Client) trigger(channels []string, event string, data interface{}, sock
 	return unmarshalledBufferedEvents(response)
 }
 
+type BatchRequest struct {
+	Batch []Event `json:"batch"`
+}
+
+func (c *Client) TriggerBatch(batch []Event) (*BufferedEvents, error) {
+	payload, err := json.Marshal(&BatchRequest{batch})
+	if err != nil {
+		return nil, err
+	}
+
+	path := fmt.Sprintf("/apps/%s/batch_events", c.AppId)
+	u := createRequestURL("POST", c.Host, path, c.Key, c.Secret, authTimestamp(), c.Secure, payload, nil, c.Cluster)
```

Comment:
Last minor comment (very personal): there 6 similar `createRequestURL` calls, which (in this file) always takes a `Client`, and always calls `authTimestamp` to get the timestamp. Might be worth creating a helper, which would also drop the dependency on `time`.